Tests/conf.json

```diff
@@ -2372,7 +2372,8 @@
         {
             "integrations": "Whois",
             "playbookID": "whois_test",
-            "fromversion": "4.1.0"
+            "fromversion": "5.0.0",
+            "timeout": 5000
```

Comment:
@ostolero - I'm increasing the timeout because of constant timeouts across all 6.8+ instances, e.g.

https://code.pan.run/xsoar/content/-/jobs/30751591#L1829
https://code.pan.run/xsoar/content/-/jobs/30751592#L1788

Reply:
I changed it to 2000 and the test playbook finished successfully.